visualization/settings/templates.py

```diff
@@ -0,0 +1,59 @@
+import os
+from pathlib import Path
+
+path = os.path.abspath(Path(__file__).parent.parent.parent)
+header = f"export PYTHONPATH='{path}:$PYTHONPATH'"
+
+generate_solution_template = (
+    header
+    + """
+python3 nemo_skills/inference/generate_solutions.py \\
```

Comment:
```suggestion
python nemo_skills/inference/generate_solutions.py \\
```
or use sys.executable like here https://github.com/Kipok/NeMo-Skills/blob/main/pipeline/summarize_results.py#L55